### DISCUSSION TITLE
Is NvChad stable?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [NvChad/NvChad#1323] [Q&A] [answered]
I was using NvChad a while ago but I stopped using it because of the amount of (breaking) changes that were happening at the time. Technically, I still use NvChad, when there was a discussion to remove the custom directory, siduck created a `customless` branch and I still use it. But I want to go back to the original NvChad. So do you think it is stable enough now? Is there a planned breaking change? Also, I use nightly neovim, is NvChad ok with it?

**Comment by siduck** [comment, marked as answer]
@ahhshm yes its way stable now, check our new site :D

**Comment by willianba** [comment]
it's also possible to create nvchad snapshots if you want to stay at a fixed version + configuration (see #1221)

**Reply by amirhhashemi** [reply]
Wow, lots of amazing features have been added to NvChad. I'll definitely try that out. Thank you.

**Reply by siduck** [reply]
@willianba snapshots will be improved & optmized soon after @LeonHeidelbach 's exam get done! It'll be possible to compress the whole nvim config + plugins + custom config into a single minified file so it'll get easy for users to save snapshots in their dotfiles repo too

**Reply by LeonHeidelbach** [reply]
@ahhshm @willianba The next milestone for me will be to make NvChadSnapshots portable. As @siduck already said, I will add functionality to NvChad that will allow users to compile their entire configuration, including all Packer plugins + versions into a single portable binary format that NvChad will be able to read and recreate snapshots from. This will allow users to easily save multiple configuration setups, stash them on a thumb drive, send them to other users or even attach them to an issue so that we can easily recreate user problems on our devices and offer better support.

**Reply by amirhhashemi** [reply]
@LeonHeidelbach That's so cool. I'm looking forward to it.
